Cargo.toml

```diff
@@ -175,6 +175,12 @@ x509-certificate = "0.23.1"
 inherits = "release"
 lto = true
 
+[profile.ci]
+inherits = "dev"
```

Comment:
I think we should resist adding more profiles. We should keep bechmarks/tests as close to release as possible.